src/Components/RecommendationsPage/RecommendationsTable.tsx

```diff
@@ -136,14 +176,53 @@ const RecommendationsTable = ({ page, perPage, setTotal }) => {
                     {rec.remediation === 'Playbook' ? <PlayIcon /> : <WrenchIcon />} {rec.remediation}
                   </Td>
                   <Td dataLabel={columnNames.run} modifier="fitContent">
-                    {rec.remediation === 'Playbook' && <Button variant="primary">{columnNames.run}</Button>}
+                    {rec.remediation === 'Playbook' && (
+                      <Button
+                        variant="primary"
+                        onClick={() => {
+                          setCurrentRule(rec);
+                          getResolutionsInfo(rec);
+                          setModalIsOpen(true);
+                        }}
+                      >
+                        {columnNames.run}
```

Comment:
Would probably use `'Create playbook'` or maybe better `'Remediate'` to have it consistent with button in advisor ... instead of run, because the playbook has to be separately executed in order for remediation to be completed. I don't like the remediate much because that also implies it will run but not sure if we should prefer consistency over minor improvement.

Reply:
and header is missing on purpose? Probably doesn't matter that much